### DISCUSSION TITLE
FileUpload Disable option

### DISCUSSION THREAD

**Discussion by mrpathi** [iTwin/iTwinUI-react#804] [Q&A]
Hi,

I want to disable the FileUpload, FileUploadTemplate @itwin/itwinui-react components. Right now there is disable probs in the component. Is there any possible way to do that?. Thanks in advance.

**Comment by mayank99** [comment]
Could you clarify what you mean by disable? And which component has the `disabled` prop?

I think you could remove the `FileUpload` wrapper when you need to disable the drag and drop functionality.

**Reply by mrpathi** [reply]
All the form components like Input, Select are having disabled prop to disable in certain conditions. I want that disabling options for FileUpload wrapper too. I need to disable file upload option for restricted users. Right now there is no disable option to do that. I hope you get my point....
